rr/gstreamer/gst_media.py

```diff
@@ -170,14 +193,43 @@ def set_triggers(self, triggers):
     def get_triggers(self):
         return self._triggers
 
+    @classmethod
+    def learn_model_config(cls, model, model_params):
+        GstMedia.model = model
+        GstMedia.model_params = model_params
+
+        GstMedia.ai_model_config = GetConfigYaml(cls.model)
+
```

Comment:
No, please pass the model and the params to the make method if possible. Saving stuff in the class struct is very dangerous.